signatures.go

```diff
@@ -20,17 +20,18 @@ import (
 	"crypto/sha256"
 	"encoding/binary"
 	"fmt"
-	"math"
 
 	"github.com/google/certificate-transparency-go/tls"
+	"github.com/google/certificate-transparency-go/x509"
 	"github.com/transparency-dev/formats/log"
-	"github.com/transparency-dev/static-ct/modules/dedup"
-	"golang.org/x/crypto/cryptobyte"
 	"golang.org/x/mod/sumdb/note"
 
 	ct "github.com/google/certificate-transparency-go"
 )
 
+const millisPerNano int64 = 1000 * 1000
```

Comment:
Aren't there approximately zero milliseconds per nanosecond, did you mean `nanosPerMilli`? :)

If so, could this be `... = time.Millisecond / time.Nanosecond`?

Reply:
Maybe! This is not new code, I'm blindly migrating it from structures.go. Fixed!

Reply:
(did you mean to also export it?)